### PR TITLE
Implement multi-stream source test adapter

### LINKED ISSUE #372: Implement multi-stream source test adapter
The adapter must create multiple video streams and send them with different source ids like `source-%d`. We can specify the parallelism and number of frames. The adapter is used for performance tests. At the end of the operation, it sends a shutdown if configured (#371).

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (denisvmedyantsev) [inline adapters/gst/gst_plugins/python/savant_rs_serializer.py]: Default should be `source-%d`

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (denisvmedyantsev) [inline adapters/gst/gst_plugins/python/savant_rs_serializer.py]: The pattern is required for multistream, it is better to check.

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (denisvmedyantsev) [inline adapters/gst/gst_plugins/python/savant_rs_serializer.py]: `source_ids = []` - Is it a valid case?

**Review by denisvmedyantsev — APPROVED** [review]
Go through my comments. No need for a second review.

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (tomskikh) [reply adapters/gst/gst_plugins/python/savant_rs_serializer.py]: Set a default value to property description.

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (tomskikh) [reply adapters/gst/gst_plugins/python/savant_rs_serializer.py]: Added a check.

File: adapters/gst/gst_plugins/python/savant_rs_serializer.py
Comment (tomskikh) [reply adapters/gst/gst_plugins/python/savant_rs_serializer.py]: Moved content of `_set_source_ids_and_zmq_sockets` to `do_start` and removed this case.

**Pull request merged**
